### DISCUSSION TITLE
Requesting for finding nearest point in two polygons

### DISCUSSION THREAD

**Discussion by dhivehi** [Turfjs/turf#2796] [Feature proposals]
It would be nice if turf.nearestPoint can also accept multiple points as 1st parameter. So that it can be used to find two closest points in two polygons.

**Comment by rowanwins** [comment]
Hi @dhivehi 

That's quite a different use case than what `nearestPoint` currently supports.

If I understand correctly you want to pass in multiple points and multiple polygons, and somehow find out which polygon is closest to each point - is that correct?

**Comment by dhivehi** [comment]
Its about finding closest points between two polygons
![default](https://user-images.githubusercontent.com/18715293/44269440-f5833b80-a24d-11e8-8bef-4894060ca481.png)

**Comment by rowanwins** [comment]
ah I see, that's clearer thanks @dhivehi
